Rakefile

```diff
@@ -36,4 +36,11 @@ task lint: %i[rubocop environment] do
   sh "yarn run lint"
 end
 
-task default: [:lint, :spec, "app:jasmine:ci"]
+desc "Jasmine"
+task :jasmine do
+  sh "which chromedriver"
+  sh "chromedriver --version"
+  sh "yarn run jasmine"
```

Comment:
I assume you'd have to run an assets precompile step before this task?

Reply:
yes, clobber and precompile required, have added them afterwards